reana_job_controller/htcondorcern_job_manager.py

```diff
@@ -95,6 +94,8 @@ def execute(self):
         """Execute / submit a job with HTCondor."""
         os.chdir(self.workflow_workspace)
         initialize_krb5_token(workflow_uuid=self.workflow_uuid)
+        globals()["htcondor"] = __import__("htcondor")
```

Comment:
- Perhaps move the import magic to the constructor, if it works OK there, because most people would expect this to happen at the class constructor stage.

- Please leave a comment why we do this, so that we recall for later. Something like "# We need to import htcondor package later during runtime after the Kerberos environment is fully initialised, otherwise..."  (The behaviour of `htcondor` package is not exactly Python-idiomatic...)